lib/utils/perf-tester.js

```diff
@@ -0,0 +1,483 @@
+(function() {
+
+  'use strict';
+
+  class PerfTester extends HTMLElement {
```

Comment:
@sorvell Was this class intended to be committed in this PR? If so, we need some docs on what it actually does and how we can use it (assuming you used it to measure the performance of the changes in this PR).

Reply:
argh, mistakenly committed; removing.